src/main/kotlin/io/kuzzle/sdk/protocol/Http.kt

```diff
@@ -220,10 +220,10 @@ open class Http : AbstractProtocol {
                         }
                     }
                     this.header("content-type", "application/json")
-                    this.body = JsonSerializer.serialize(payload)
+                    this.setBody("JsonSerializer.serialize(payload)")
```

Comment:
```suggestion
                    this.setBody(JsonSerializer.serialize(payload))
```
This should not be stringified